CI/cdash-url.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
 dir=$1
+cdash_url=$2
 tag=$(head -1 $dir/TAG)
 Done=$(cat $dir/$tag/Done.xml)
 buildID=$(echo $Done | grep -o -P '(?<=buildId>).*(?=</build)')
-URL=https://cdash.jcsda.org/buildSummary.php?buildid=$buildID
+URL=$cdash_url/viewTest.php?buildid=$buildID
```

Comment:
Is this change intended? `buildSummary.php` to `viewTest.php`?

Reply:
I prefer the older one, because we do have some compiled C++ and Fortran code in ioda-converters. We want to track build warnings for Ron's development.

Reply:
It should  be `buildSummary`.